shannon/testnet/genesis.json

```diff
@@ -0,0 +1,742 @@
+                  "key": "MTpNY7QZepeUlTW28YhCAv3f53N9XS2lb9bqF0PN3Sk="
+                },
+                "value": {
+                  "denom": "upokt",
+                  "amount": "1000000000000000"
+                }
+              }
+            ],
+            "memo": "ab57afd72103ad66f02bd9ad271435fba49f2326@192.168.88.58:26656",
+            "timeout_height": "0",
+            "extension_options": [],
+            "non_critical_extension_options": []
+          },
+          "auth_info": {
+            "signer_infos": [
+              {
+                "public_key": {
+                  "@type": "/cosmos.crypto.secp256k1.PubKey",
+                  "key": "A4CdzV1JUwNGtQZYYlPZOFoMWxsEpFTyQcTh0TtibdYS"
+                },
+                "mode_info": {
+                  "single": {
+                    "mode": "SIGN_MODE_DIRECT"
+                  }
+                },
+                "sequence": "0"
+              }
+            ],
+            "fee": {
+              "amount": [],
+              "gas_limit": "200000",
+              "payer": "",
+              "granter": ""
+            },
+            "tip": null
+          },
+          "signatures": [
+            "bAuD8GfIW1A7TisM8wzYz8bg+ftrHA7GaeA0++bbg/MIle9I3AloyyH5WBOoKRuOzEXO07dfgg0JS3yaZ9EB/g=="
+          ]
+        }
+      ]
+    },
+    "gov": {
+      "constitution": "",
+      "deposit_params": null,
+      "deposits": [],
+      "params": {
+        "burn_proposal_deposit_prevote": false,
+        "burn_vote_quorum": false,
+        "burn_vote_veto": true,
+        "expedited_min_deposit": [
+          {
+            "amount": "50000000",
+            "denom": "stake"
+          }
+        ],
+        "expedited_threshold": "0.667000000000000000",
+        "expedited_voting_period": "86400s",
+        "max_deposit_period": "172800s",
+        "min_deposit": [
+          {
+            "amount": "10000",
+            "denom": "upokt"
+          }
+        ],
+        "min_deposit_ratio": "0.010000000000000000",
+        "min_initial_deposit_ratio": "0.000000000000000000",
+        "proposal_cancel_dest": "",
+        "proposal_cancel_ratio": "0.500000000000000000",
+        "quorum": "0.334000000000000000",
+        "threshold": "0.500000000000000000",
+        "veto_threshold": "0.334000000000000000",
+        "voting_period": "172800s"
+      },
+      "proposals": [],
+      "starting_proposal_id": "1",
+      "tally_params": null,
+      "votes": [],
+      "voting_params": null
+    },
+    "group": {
+      "group_members": [],
+      "group_policies": [],
+      "group_policy_seq": "0",
+      "group_seq": "0",
+      "groups": [],
+      "proposal_seq": "0",
+      "proposals": [],
+      "votes": []
+    },
+    "mint": {
+      "minter": {
+        "annual_provisions": "0.000000000000000000",
+        "inflation": "0.130000000000000000"
+      },
+      "params": {
+        "blocks_per_year": "6311520",
```

Comment:
So what's our current block time?

Reply:
I don't know what that number comes from or if it is used at all.

Block times are specified in validator configuration files. Currently `300s`.

`6311520` blocks per year is ~= 5s/block:
<img width="765" alt="image" src="https://github.com/user-attachments/assets/319e110c-2352-4b36-b68c-34b67e8f572e">

Reply:
Ah, this is `mint` module. I suppose this is used to reward validators. IIRC, we're not going to inflate the network with validator rewards, only with relay service.

Reply:
This is nuanced answer. You're both right & wrong at the same time but we'll cover that offline.